src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const lines = stylesString.trim().split('\n');
+  const cleanedLines = lines.filter((line) => line.trim() !== '');
+
+  const cleanedString = cleanedLines.join('\n');
+  const declarations = cleanedString.split(';');
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. You can chain the split(';') method directly after the join('\n') without creating an intermediate 'cleanedString' variable.